### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by brian-smith723**
Changes the structure of the Navbar.

### CODE REVIEW

File: ooiui/static/js/views/common/NavbarView.js [hunk 1]
```diff
@@ -60,8 +60,10 @@ var NavbarView = Backbone.View.extend({
     // Messages only appear to logged in users
     if(ooi.login.loggedIn()) {
         this.$el.find('#navbar-menus').append(this.templates.logged_in_nav_items());
+        this.$el.find('#Science').hide();
         //this.$el.find('#navbar-menus').append(this.messageView.el);
     }
     this.$el.find('#navbar-menus').append(this.dropdownUserView.el);
+   
   }
-});
```
Comment (DanielJMaher) [inline ooiui/static/js/views/common/NavbarView.js]: Is this a concern?

Comment (birdage) [inline ooiui/static/js/views/common/NavbarView.js]: it got moved to the line below

**Comment by DanielJMaher** [comment]
@birdage So you think this is good?

File: ooiui/static/js/views/common/NavbarView.js (hunk 1 above)
Comment (brian-smith723) [inline ooiui/static/js/views/common/NavbarView.js]: I think Dan is talking about line 63 this.$el.find('#Science').hide();

My answer: I have no idea.

We already have a Logged in partial. So I added the logged in science drop down to the log in partial and hide the other one.  When you log out the logged in partial goes away and you see the original link. I"m not adding a class just hiding it, if a user is logged in.

Comment (DanielJMaher) [inline ooiui/static/js/views/common/NavbarView.js]: No, Andy was right.  I jsut wanted to make sure that automatic Red Flag was addressed.

**Pull request merged**